test/functional/DDR_Test/playlist.xml

```diff
@@ -120,18 +119,136 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 		<testCaseName>testDDRExt_Callsites_win</testCaseName>
```

Comment:
We mark test on Windows as part of sanity group and the non-Windows for the extended group.

Reply:
Ok, but why is Windows treated specially with respect to this test?

Reply:
It is most likely treated like that because we carried over the levels directly as they were from the old internal system that would have tagged that platform as 'sanity' / gold / blocking.  There is no way for us to know 'why' that would have been done.  

If I were to guess, it was likely done to choose 1 platform out of the bunch to watch to for failures (and that could block a build), with the other platforms moved out to extended, to keep the sanity execution time shorter.  Since we now look at both sanity & extended failures more closely in the openj9 project, I think it is fine to collapse all platforms into 1 test target tagged as extended.